.env.example

```diff
@@ -0,0 +1,6 @@
+# Example env file. Copy this file to .env and fill with your corresponding LeanPlum key values. 
```

Comment:
does `bundle exec rspec` automatically load this file? that would be dope

Reply:
it does. 
You can also have .env.development, .env.test, etc.... with env specific overrides from the base .env file.